src/cow-react/modules/limitOrders/pure/DeadlineSelector/index.tsx

```diff
@@ -145,4 +182,67 @@ export function DeadlineSelector(props: DeadlineSelectorProps) {
   )
 }
 
-// write bash cron job to run this script every day at 00:00
+function _limitDateString(date: Date | number): string {
```

Comment:
I think it makes sense to move all utility functions to separated file, like a `DeadlineSelector/utils.ts`